### PR TITLE
Writing page > Major additions/changes

### PR DESCRIPTION
**Pull request by geurts**
Major additions and changes to Writing page documentation authored by @nicolebezaire. Some style changes in the scope of the Writing page. Also some style tweaks to the global styles in `page-content.css` that should not affect component pages.

### CODE REVIEW

**Review by nicolebezaire — CHANGES_REQUESTED** [review]
Yey! SO rewarding to see my updates to the Writing style guide go to implementation. See my revision requests over in our slack thread :)

File: pages/_includes/css/page-content.css [hunk 1]
```diff
@@ -21,14 +21,14 @@
...
-	margin: 1rem 0;
+	margin: 1.5rem 0;
 }
 @media (max-width: 1200px) {
 	.d2l-daylight-best-practices {
 		flex-direction: column;
 	}
 }
-div.d2l-daylight-dos, div.d2l-daylight-donts {
+.d2l-daylight-dos, .d2l-daylight-donts {
```
[the remark this answers is not in the document]
Comment (geurts) [reply pages/_includes/css/page-content.css:31]: Doesn't seem to be necessary, but happy to re-add it if you think it's better to be more specific?

File: pages/style-elements/writing.md [hunk 2]
```diff
@@ -7,132 +7,188 @@ eleventyNavigation:
 ---
 
 <style>
+  @font-face {
+    font-family: ForQuotationMarks;
+    src: url("../_includes/BodoniModa-VariableFont_opsz,wght.ttf");
```
[the remark this answers is not in the document]
Comment (geurts) [reply pages/style-elements/writing.md:12]: It's how the file was named by google fonts. I thought about simplifying to "BodoniModa.ttf" but thought it was probably better to leave it. Should I simplify it then?

[the remark this answers is not in the document]
Comment (geurts) [reply pages/style-elements/writing.md]: Was able to remove the web font anyway, yey :)

File: pages/style-elements/writing.md [hunk 3]
```diff
@@ -7,132 +7,188 @@ eleventyNavigation:
...
 # Writing
 
-```"Every word and letter in an interface matters. Good writing is good design. Text is ultimately interface, and it’s us, the designers, who are the copywriters shaping this information."```
-<div style="text-align:end;">– Viljami Salminen</div>
+<blockquote>
+  <p>
+    Every word and letter in an interface matters. <em>Good writing is good design</em>. Text is ultimately interface, and it’s us, the designers, who are the copywriters shaping this information.
+  </p>
+  <p>
+    <div style="text-align:start;font-size:.9em;color:var(--d2l-color-galena);">Viljami Salminen</div>
```
[the remark this answers is not in the document]
Comment (geurts) [reply pages/style-elements/writing.md]: Done!

File: pages/style-elements/writing.md [hunk 4]
```diff
@@ -142,85 +198,311 @@ If it is a **branded** product term or proper noun, use title case. *Human names
...
+| Break link to | Terminate the link from the copy to the source. The copy remains a copy of the original, but it no longer mirrors its properties to a source. |
+| Locked | Applies to object properties that maintain a link to the source. If these properties change on the source, they cascade to all linked copies. |
+| Unlocked | Applies to object properties that do not maintain a link to the source (such as due dates). If these properties change on the source, it does not change the linked copies. |
+| Share | Initiate allowing access to an object, to other people. |
+| Shared | The status of an object that is available to others. Others would have to copy the object before they can use it. |
+
+<p style="margin-top:-12px !important">For further detail and definition, check out <a href="https://desire2learn.atlassian.net/wiki/spaces/PD/pages/1487310133/Activity+Re-use+and+Sharing+Language">Activity Reuse and Sharing Language</a> on Confluence.</p>
```
[the remark this answers is not in the document]
Comment (geurts) [reply pages/style-elements/writing.md]: I actually removed that, yeah. Strange that originally it addressed a gap that was too large, not sure what changed but happy to have it removed 🤷

File: pages/_includes/css/page-content.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (margaree) [reply pages/_includes/css/page-content.css:31]: If it's not necessary then I'd say leave it out

**Review by margaree — APPROVED** [review]

**Review by nicolebezaire — APPROVED** [review]
👍

**Pull request merged**